build_tools/github_actions/should_ci_run.py

```diff
@@ -16,7 +16,7 @@
 import subprocess
 import sys
 
-SKIP_CI_LABEL = "skip-ci"
+SKIPC_CI_TAG = "skip-ci"
```

Comment:
extra character in this variable name?

```suggestion
SKIP_CI_TAG = "skip-ci"
```